fraud-detection-new-isoscape/origin_validation.py

```diff
@@ -7,7 +7,7 @@
 
 """Global Variable definition"""
 
-ISOSCAPES_EE_PATH = 'projects/river-sky-386919/assets/isoscapes'
+ISOSCAPES_EE_PATH = 'projects/timberid-prd/assets/isoscapes'
```

Comment:
branch main corresponds to river-sky-386919 it shouldn't be changed to timberid-prd

Reply:
we should not have different code per branch -- thats going to break when someone unwittingly merges in and changes it.  lets discuss monday